BeDoLaFinal/151Final
# Created gameObjects.h and class for screens

Created 'gameObjects.h' file to house game objects classes; built class for screens; tested in main.cpp
Created folder for images
Added compile and run command for testing to .gitignore

***TO TEST***
open XLaunch as normal
run      $ g++ *.cpp -o test -lsfml-system -lsfml-window -lsfml-graphics
then    $ ./test

Window should open the Battleship logo... left click to display "youWon" image and right click to display "youLost" image
